allennlp/data/instance.py

```diff
@@ -60,3 +60,9 @@ def pad(self, padding_lengths: Dict[str, Dict[str, int]]) -> Dict[str, List[nump
 
     def fields(self):
         return self._fields
+
+    def get_field(self, name: str):
```

Comment:
This unnecessarily clutters the API when we can use `self.fields()["name"]` instead. It also stops us accidentally using this in allennlp internals where we assume that an `Instance` has a key or something.